doctests/dt_list.py

```diff
@@ -0,0 +1,324 @@
+# EXAMPLE: list_tutorial
+# HIDE_START
+import redis
+
+r = redis.Redis(decode_responses=True)
+r.delete("bikes:repairs")
```

Comment:
Please wrap those two lines in the REMOVE_START/END block to hide this cleanup code from the users.